xml/art_admin_slemicro_cockpit.xml

```diff
@@ -125,945 +139,2118 @@
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_dataflow_view.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_dataflow_view.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Cockpit storage</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+  </section>
+  <section xml:id="sec-disk-formatting">
+   <title>Managing file systems</title>
+   <para>
+    The <guimenu>Filesystems</guimenu> view enables you to create a partition
+    table and to format or mount file systems. You can sort the mounted
+    partition according to <guimenu>Name</guimenu> or <guimenu>Mount
+    point</guimenu>. Click on the particular file system to open the details
+    window.
+   </para>
+   <figure xml:id="fig-storage-filesystems">
+    <title>File systems details</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_filesystems.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_filesystems.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Cockpit storage</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+   <para>
+    To format the partition, click <guimenu>Format</guimenu> next to the
+    particular partition description to open the format window.
+   </para>
+   <figure xml:id="fig-formatting-partitions">
+    <title>Formatting partitions</title>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata  fileref="cockpit_format_partitions.png" width="100%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="cockpit_format_partitions.png" width="100%"/>
+     </imageobject>
+     <textobject role="description"><phrase>Format partition</phrase>
+     </textobject>
+    </mediaobject>
+   </figure>
+   <para>
+    You can specify the following details:
+   </para>
+   <variablelist xml:id="list-format-partition">
+    <title>Formatting partitions parameters</title>
+    <varlistentry>
+     <term>Name</term>
+     <listitem>
+      <para>
+       enter a unique name of the partition.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Type</term>
+     <listitem>
+      <para>
+       select the file system. BTRFS is mandatory for the <literal>/</literal>
+       partition.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Mount point</term>
+     <listitem>
+      <para>
+       this is a mandatory option. Specify to which directory the partition
+       will be mounted.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Mount options</term>
+     <listitem>
+      <para>
+       in the <guimenu>Custom mount options</guimenu> text field you can enter
+       a comma-separated list of options. For common the options, refer to
```

Comment:
I'm not a native english speaker but "common the options" doesn't sound correct to me.

Reply:
Oh no! But I guess Daria (our editor) will spot it too :)